nifi-nar-bundles/nifi-social-media-bundle/nifi-twitter-processors/pom.xml

```diff
@@ -35,6 +35,21 @@
             <artifactId>nifi-utils</artifactId>
             <version>1.17.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>com.twitter</groupId>
+            <artifactId>twitter-api-java-sdk</artifactId>
+            <version>1.1.2</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.json</groupId>
+                    <artifactId>json</artifactId>
+                </exclusion>
```

Comment:
This dependency does not appear to included, so the exclusion can be removed.

Reply:
Thanks for the comments, removing it!